ScoreSaber/UI/Elements/Profile/ProfileDetailView.cs

```diff
@@ -134,6 +237,14 @@ protected void Parsed() {
             PanelView.ImageSkew(ref _profileTop) = 0f;
 
             modalPic.material = Plugin.NoGlowMatRound;
+            mapImagePresence.material = Plugin.NoGlowMatRound;
+            mapPresence.gameObject.SetActive(false);
+            if(_beatmapDownloader == null) {
+                Plugin.Log.Error($"BeatmapDownloader is null, install a mod to [APP] that injects IScoreSaberBeatmapDownloader");
+            }
+            // load sprites
+            UnityMainThreadTaskScheduler.Factory.StartNew(() => BeatSaberMarkupLanguage.Utilities.LoadSpriteFromAssemblyAsync("ScoreSaber.Resources.Online.png")).ContinueWith(x => { _onlineSprite = x.Result.Result; });
```

Comment:
We reference these strings quite frequently (later on in the file too). We should have one source of truth for them.

Reply:
Made constants